### PR TITLE
34 admin panel navigation

### PR DESCRIPTION
**Pull request by shinymew2**
Unter dem Pfad /admin/users kann man die Navbar testen

### CODE REVIEW

File: src/components/admin/NavbarAdmin.vue [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        :title="$t('admin.navbar.manage_task')"
+        @click="goToTask"
+      ></v-list-item>
+      <v-list-item
+        prepend-icon="mdi-file-tree"
+        :title="$t('admin.navbar.manage_course')"
+        @click="goToCourse"
+        disabled
+      ></v-list-item>
+    </v-list>
```
Comment (hwgn) [inline src/components/admin/NavbarAdmin.vue:26]: Could you add a "Leave" button to the bottom of the drawer? That way you don't get stuck in the admin area

File: src/components/admin/NavbarAdmin.vue [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<template>
+  <v-navigation-drawer permanent>
```
Comment (hwgn) [inline src/components/admin/NavbarAdmin.vue:2]: Is there some way to make it hide on smaller devices and only open on request?

**Review by hwgn — CHANGES_REQUESTED** [review]
Looks really nice overall!

File: src/views/admin/TaskManagementView.vue [hunk 3]
```diff
@@ -1,21 +1,18 @@
 <template>
-  <!-- Admin Panel Header/Menu -->
-  <SkipToContent/>
   <NavbarAdmin/>
```
Comment (hwgn) [inline src/views/admin/TaskManagementView.vue:2]: The navbar is currently covering the content a little

**Review by hwgn — CHANGES_REQUESTED** [review]

**Review by shinymew2 — COMMENTED** [review]
Looks good.

**Review by hwgn — APPROVED** [review]
LGTM

**Pull request merged**